modules/contact/test/tests/multiple_contact_pairs/tests

```diff
@@ -1,13 +1,36 @@
 [Tests]
   issues = '#18022'
-  design = 'syntax/Contact/index.md MechanicalContactConstraint.md'
+  design = 'syntax/Contact/index.md MechanicalContactConstraint.md ContactPressureAux.md'
   [multiple_pairs]
     type = Exodiff
     input = 'multiple_pairs.i'
     exodiff = 'multiple_pairs_out.e'
     abs_zero = 1.0e-8
     rel_err = 1.0e-5
     cli_args = "Postprocessors/active=''"
-    requirement = 'Contact module action shall allow for multiple contact pairs when selecting node-face formulation.'
+    requirement = 'Contact module action shall allow for multiple contact pairs when selecting '
```

Comment:
`'The system shall set up node-face interactions between multiple surface pairs using the Contact action'`